tests/unit/test.program.js

```diff
@@ -242,13 +256,37 @@ describe('program.Program', () => {
       });
   });
 
+  it('checks for updates automatically', () => {
+    const root = path.join(__dirname, '..', '..');
+    const handler = spy();
+    const checkForAutomaticUpdates = sinon.stub();
+    const program = new Program(['run'])
+    .command('run', 'some command', handler);
```

Comment:
This should be indented two spaces to show that it's a continuation of the call up above.